velox/functions/prestosql/Arithmetic.h

```diff
@@ -198,6 +198,30 @@ struct Log10Function {
   }
 };
 
+template <typename T>
```

Comment:
why register the function in sparksql folder and implement it in prestosql folder